libraries/points/README.md

```diff
@@ -1,13 +1,188 @@
+  reason: String! @string(maxLength: 100)
+}
+```
+
+## Default Hosted Node (Testing Purposes Only)
+
+The points library uses a default node endpoint (for test purposes only) that has the standard types and interfaces deployed onto it. Unless otherwise indicated by the developer, the library will use this hosted node.
+
+For developers who want to use the library with bespoke type extensions of the standard interfaces, they will need to deploy the library's composite onto their own node, as well as the model extensions they wish to use.
+
+Note: this hosted node is designed to make it easier for developers to quickly experiment with this library, but is NOT intended for production purposes. If you use this default node in production, your data may be lost.
+
+## Usage
+
+Instantiating a PointsWriter/PointsReader from seed:
+
+```TypeScript
+import { PointsReader, PointsWriter } from '@ceramic-solutions/points'
+import { getAuthenticatedDID } from '@ceramic-solutions/key-did'
+import { fromString } from 'uint8arrays'
+
+// import environment variable
+const CERAMIC_PRIVATE_KEY: string = process.env.CERAMIC_PRIVATE_KEY || ''
+
+// transform private key to Uint8Array
+const key = fromString(CERAMIC_PRIVATE_KEY, 'base16') as Uint8Array
+
+// generate issuer for reader context
+const issuer = await getAuthenticatedDID(key)
+
+// instantiate a PointsWriter using default hosted node
+const writer = await PointsWriter.fromSeed({
+    seed: key
+    // ceramic: <your node endpoint> (if using custom node)
+  })
+
+// instantiate a PointsReader and specify application as issuer
+const reader = PointsReader.create({
+    issuer: issuer.id,
+    // ceramic: <your node endpoint> (if using custom node)
+  })
+```
+
+Reading and writing to aggregation documents for a recipient:
+
+```TypeScript
+
+// recipient must be did:pkh format
+const { amount, recipient } = req.body
+
+// using default `SimplePointsAggregation` type
+const doc = await reader.loadAggregationDocumentFor([recipient])
+const totalPoints = doc.content.points
+
+// create if does not exist
+if (!doc) {
+      await writer.setPointsAggregationFor([recipient], amount, {
+        recipient,
+        points: amount,
+        date: new Date().toISOString()
+      }) 
+    } else {
+      const updatedDoc = await writer.updatePointsAggregationFor(
+        [recipient],
+        (content) => {
+          return {
+            points: content ? content.points + amount : amount,
+            date: new Date().toISOString(),
+            recipient
+          }
+        },
+      )
+    }
+
+// allocate points to track history
+await writer.allocatePointsTo(recipient, amount, {
+    recipient,
+    points: amount,
+    date: new Date().toISOString()
+})
+```
+
+Note: developers who wish to use the allocation and aggregation functionality together must control for keeping the totals for each in sync within their application logic - this is not automatically accounted for in the library.
+
```

Comment:
do you want to link here to pullToPoints as an example of how?  i can do that later in a separate PR.   its currently at https://github.com/ceramicstudio/pullToPoints could be renamed